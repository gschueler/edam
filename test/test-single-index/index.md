% Test Single Index

Test single page with an index
