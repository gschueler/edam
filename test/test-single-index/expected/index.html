<!DOCTYPE html>
<html>
<head>
  <title>Test Single Index</title>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta name="generator" content="pandoc" />
  <link rel="stylesheet" href="style.css" type="text/css" />
</head>
<body>


<nav class="page top">
    <ul>
        <li class="current"><a href="index.html">Test Single Index</a></li>
        
        
        
    </ul>
</nav>
<div id="docbody">
<h1 class="title">Test Single Index</h1>
<p
>Test single page with an index</p
>
</div>

<nav class="page bottom">
    <ul>
        <li class="current"><a href="index.html">Test Single Index</a></li>
        
        
        
    </ul>
</nav>

</body>
</html>